### DISCUSSION TITLE
Start tour hidden?

### DISCUSSION THREAD

**Discussion by schvenk** [elrumordelaluz/reactour#475] [Q&A]
Is there a way to start the tour hidden—that is, initially it's not visible and the I can call `setIsOpen()` to open it? (Underlying reason is I'd like a slight delay on showing it after the page loads, so I suppose if there's a way to do that directly that's great, too.)

Thanks!

**Comment by elrumordelaluz** [comment]
Hi @schvenk, I think I am not fully understand the point, isn't enough to call `setIsOpen()` later?

**Reply by schvenk** [reply]
Well...my tour automatically loads as soon as the app does. So in theory yes, I could call `setIsOpen(false)`, but there's not an obvious point at which to do that. I mean, I could use a `setTimeout()` and wait, say, 500ms, but that's messy.

**Reply by elrumordelaluz** [reply]
could you share a minimal reproduction in a sandbox? in general, the Tour is `open: false` and you have to trigger the state change to open it, like in [these examples](https://reactour.vercel.app/#tourprovider)

**Reply by schvenk** [reply]
Actually I hadn't realized it was supposed to start with `open: false` and the minute you said that I went back and audited my code and found the issue. Thanks.
